main.go

```diff
@@ -66,13 +69,13 @@ func saveResults(out chan DnsRecordResult) {
 		panic(err)
 	}
 
-	ins, err := db.Prepare("INSERT INTO dns_records (`domain`, `type`, `value`) VALUES (?, ?, ?)")
+	ins, err := db.Prepare("INSERT INTO dns_records (`domain`, `type`, `value`, `ip_dec`) VALUES (?, ?, ?, ?)")
 	if err != nil {
 		panic(err)
 	}
 
 	for result := range out {
-		ins.Bind(result.Domain, dns.TypeToString[result.RecordType], result.Value)
+		ins.Bind(result.Domain, result.RecordType, result.Value, result.IPDec.String())
```

Comment:
On line 15 you changed record type to string but `result.RecordType` is `uint16`, shouldn't this line be using `result.RecordType`

Also note that previously `DnsRecordResult.RecordType` was set to `uint16` and I was saving `dns.TypeToString[result.RecordType]` which doesn't seem correct